pkg/ingestor/ingestcli/interface.go

```diff
@@ -24,6 +24,7 @@ import (
 // WriteRequest is the request to write KV to storage layer.
 type WriteRequest struct {
 	ClusterID string
+	ChunkID   int64
```

Comment:
WriteClient should maintain this `ChunkID`, after each success call of it, it increase it internally. As `WriteClient` do retry internally, the outer logic will retry the whole task if any write chunk fails